src/nni_manager/training_service/common/gpuData.ts

```diff
@@ -58,3 +58,11 @@ export class GPUSummary {
         this.gpuInfos = gpuInfos;
     }
 }
+
+export const GPU_COLLECTOR_FORMAT: string = 
```

Comment:
How about GPU_INFO_COLLECTOR_FORMAT

Reply:
You should also consider if the script works on Windows. 
However, it's not urgent for this hot fix. You can update the code for Windows after merging.

Reply:
Changed to `GPU_INFO_COLLECTOR_FORMAT`. We may change the format to support windows in another PR.